### PR TITLE
Add avatar component

### PR DESCRIPTION
**Pull request by aesha-spreadd**
Add avatar component
1) Circle avatar
Screenshot 2023-12-26 at 10 14 51 AM

2) Square Avatar
Screenshot 2023-12-26 at 10 15 05 AM

- [x]  Checking the code is duplicate or not?
- [x]  Is these code has bug
- [ ]  Is code is check by shubhum sir or ankit sir?
- [ ]  Are you going to merge this or not?

### CODE REVIEW

File: src/Avatar/Avatar.Styled.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import styled, { css } from 'styled-components';
+
+const sizes = {
+  small: { width: '3.4rem', height: '3.4rem' },
```
Comment (ankitadrixus08) [inline src/Avatar/Avatar.Styled.js]: @aeshapatel025 how many px equals to 3.4rem here?

File: src/Avatar/Avatar.Styled.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  align-items: center;
+  ${({ type }) => `
+    width: ${sizes[type]?.width};
+    height: ${sizes[type]?.height};
+  `}
+`;
+
+export const StyledAvatarComponent = styled.div`
+  display: flex;
+  flex-direction: row;
```
Comment (ankitadrixus08) [inline src/Avatar/Avatar.Styled.js]: you don't need to give this @aeshapatel025

File: src/Avatar/Avatar.Styled.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  gap: 1rem;
+  justify-content: center;
```
Comment (ankitadrixus08) [inline src/Avatar/Avatar.Styled.js]: where is align item? you don't need here? @aeshapatel025

File: src/Avatar/Avatar.Styled.js [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+      overflow: hidden;
+      margin: 0 5px;
+      ${avatarStyles}
+    }
+  `}
+`;
+
+export const AvatarContainer = styled.div`
+  display: flex;
+  flex-direction: row;
```
Comment (ankitadrixus08) [inline src/Avatar/Avatar.Styled.js:30]: same as above @aeshapatel025

File: src/Avatar/Avatar.Styled.js (hunk 2 above)
Comment (aesha-spreadd) [reply src/Avatar/Avatar.Styled.js]: I have remove it from StyledAvatarComponent

File: src/Avatar/Avatar.Styled.js (hunk 1 above)
Comment (aesha-spreadd) [reply src/Avatar/Avatar.Styled.js]: 54.4 px but i have removed it and add px here

File: src/Avatar/Avatar.Styled.js (hunk 3 above)
Comment (aesha-spreadd) [reply src/Avatar/Avatar.Styled.js]: I have remove  display: flex;
  flex-direction: row;
  gap: 1rem;
  justify-content: center; from here and i only put it in the AvatarContainer

File: src/Avatar/Avatar.Styled.js (hunk 4 above)
Comment (aesha-spreadd) [reply src/Avatar/Avatar.Styled.js:30]: Add in AvatarContainer
  display: flex;
  flex-direction: row;
  gap: 1rem;
  justify-content: center;
  align-items: center;

File: src/Avatar/Avatar.Styled.js [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+  ${({ type }) => `
+    width: ${sizes[type]?.width};
+    height: ${sizes[type]?.height};
+  `}
+`;
+
+export const StyledAvatarComponent = styled.div`
+  ${({ variant }) => css`
+    img {
+      border-radius: ${variant === 'square' ? '0.5rem' : '50%'};
```
Comment (tarakhant1297) [inline src/Avatar/Avatar.Styled.js]: @aeshapatel025 use shape insted of variant

File: src/Avatar/Avatar.Styled.js [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+export const AvatarContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  gap: 1rem;
+  justify-content: center;
+  align-items: center;
+`;
+
+export default StyledAvatarComponent;
```
Comment (tarakhant1297) [inline src/Avatar/Avatar.Styled.js]: @aeshapatel025 remove this empty styled

File: src/Avatar/Avatar.js [hunk 7]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { StyledAvatarComponent } from './Avatar.Styled';
+
+const Avatar = ({ type, img, variant }) => {
```
Comment (tarakhant1297) [inline src/Avatar/Avatar.js]: @aeshapatel025 use `size` instead of `type`

**Review by tarakhant1297 — CHANGES_REQUESTED** [review]

**Review by ankitadrixus08 — APPROVED** [review]

**Pull request merged**